server/src/main/java/org/spine3/server/stand/Stand.java

```diff
@@ -0,0 +1,663 @@
+import org.spine3.base.Responses;
+import org.spine3.client.EntityFilters;
+import org.spine3.client.EntityId;
+import org.spine3.client.EntityIdFilter;
+import org.spine3.client.Query;
+import org.spine3.client.QueryResponse;
+import org.spine3.client.Subscription;
+import org.spine3.client.Target;
+import org.spine3.protobuf.AnyPacker;
+import org.spine3.protobuf.KnownTypes;
+import org.spine3.protobuf.Timestamps;
+import org.spine3.protobuf.TypeUrl;
+import org.spine3.server.aggregate.AggregateRepository;
+import org.spine3.server.entity.Entity;
+import org.spine3.server.entity.EntityRepository;
+import org.spine3.server.entity.Repository;
+import org.spine3.server.storage.EntityStorageRecord;
+import org.spine3.server.storage.StandStorage;
+import org.spine3.server.storage.memory.InMemoryStandStorage;
+
+import javax.annotation.CheckReturnValue;
+import javax.annotation.Nullable;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+import java.util.concurrent.Executor;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static com.google.common.collect.Maps.newHashMap;
+
+/**
+ * A container for storing the lastest {@link org.spine3.server.aggregate.Aggregate} states.
+ *
+ * <p>Provides an optimal way to access the latest state of published aggregates for read-side services.
+ * The aggregate states are delivered to the instance of {@code Stand} through {@link StandFunnel}
+ * from {@link org.spine3.server.aggregate.AggregateRepository} instances.
+ *
+ * <p>In order to provide a flexibility in defining data access policies, {@code Stand} contains only the states
+ * of published aggregates. Please refer to {@link org.spine3.server.aggregate.Aggregate} for publication description.
+ *
+ * <p>Each {@link org.spine3.server.BoundedContext} contains the only instance of {@code Stand}.
+ *
+ * @author Alex Tymchenko
+ */
+public class Stand implements AutoCloseable {
+
+    /**
+     * Persistent storage for the latest {@link org.spine3.server.aggregate.Aggregate} states.
+     *
+     * <p>Any {@code Aggregate} state delivered to this instance of {@code Stand} is persisted to this storage.
+     */
+    private final StandStorage storage;
+
+    /**
+     * Manages the subscriptions for this instance of {@code Stand}.
+     */
+    private final StandSubscriptionRegistry subscriptionRegistry = new StandSubscriptionRegistry();
+
+
+    /** An instance of executor used to invoke callbacks */
+    private final Executor callbackExecutor;
+
+    /** The mapping between {@code TypeUrl} instances and repositories providing the entities of this type */
+    private final ConcurrentMap<TypeUrl, EntityRepository<?, ? extends Entity, ? extends Message>> typeToRepositoryMap = new ConcurrentHashMap<>();
```

Comment:
This is the only declaration of this map. And according to the idea, it _has_ to stay inside of this class, not been passed as a parameter elsewhere.

Besides, now it is clear it is a concurrency-friendly structure, which is important here.

I'd rather not create one more wrapper, since less code seems to be better.